### ISSUE TITLE
Hooked Jaws Fishing Hook Cannot Be Removed by Action

### ISSUE THREAD

**Issue by ImogenOC** [NovaSector/NovaSector#5571]
### Client Version:

516.1659

### Issue Summary:

When using the hooked jaws hook on a player, the embedded hook can be actioned on but not actually removed, resulting in a permanent movespeed slowdown requiring admin intervention.

### Round ID:

[13961](https://scrubby.melonmesa.com/round/13961)

### Test Merge Information:

- [[Balance] Blocks NPC's and Explosions from removing fishing uniques. Adds regen to items that dont have it, adds skeleton keys to lavaland fish spots.](https://github.com/NovaSector/NovaSector/pull/4916)
- [[SEMI-MODULAR] Neuroware 

### Reproduction Steps:

1. Cast hook on sapient carbon
2. Attempt to remove hook
3. Channel hook removal
4. Fail to remove hook while getting successfully removed text
5. Lament
6. See step
